docs/src/extremevalue/generalities.md

```diff
@@ -0,0 +1,131 @@
+
+$$C(u_1, u_2)=\exp\{-\ell(\log(u_1),\log(u_2))\},$$
+
+or through a convex function $A: [0,1] \to [1/2, 1]$ satisfying $\max(t, t-1)\leq A(t) \leq 1,$ called its Pickands dependence function:
+
+$$C(u_1,u_2)=\exp\left\{\log(u_1u_2)A\left(\frac{\log(u_1)}{\log(u_1u_2)}\right)\right\},$$
+
+In the context of bivariate extreme value copulas, the functions $\ell$ and $A$ are related as follows:
+
+$$\ell(u_1,u_2)=(u_1+u_2)A\left(\frac{u_1}{u_1+u_2}\right).$$
+
+Therefore, in this implementation, it is sufficient to provide the Pickands dependence function $A$ to construct the implementation structure of an extreme value copula.
+
+In this package, there is an abstract type [`ExtremeValueCopula`](@ref) that provides a foundation for defining extreme value copulas. Many extreme value copulas are already implemented for you! See [this list](@ref available_extreme_models) to get an overview.
+
+If you do not find the one you need, you may define it yourself by subtyping `ExtremeValueCopula`. The API does not require much information, which is really convenient. Only a method for the pickand function `A(C::ExtremeValueCopula) = ...` is required. By providing this functions, you can easily create a new extreme value copula that fits your specific needs:
+
+```julia
+struct MyExtremeValueCopula{P} <: ExtremeValueCopula{P}
+    θ::P
+end
+
+A(C::ExtremeValueCopula, t) = (t^C.θ + (1 - t)^C.θ)^(1/C.θ) # This is the Pickands function of the Logistic (Gumbel) Copula
+```
+
+!!! info "Nomenclature information"
+    We have called `A()` the Pickands function, which is necessary for constructing the Extreme Value Copula. This binding is very generic and thus not exported from the package, you can use it through `Copulas.A()` and/or by importing it.
+
+# Advanced Concepts
+
+Here, we present some important concepts from the theory of extreme value copulas that are useful for the development of this package.
+
+Let $(X,Y) \sim C$ where $C$ is a bivariate extreme value copula. We have the following results:
+
+> **Proposition 1 (Ghoudi 1998 [ghoudi1998proprietes](@cite)):** Let $(X, Y) \sim C$, where $C$ is an extreme value copula. The joint distribution of $X$ and $Z = \frac{\log(X)}{\log(XY)}$ is given by:
+>
+> $$P(Z \leq z, X \leq x) =G(z,x)=\left(z + z(1-z)\frac{A'(z)}{A(z)}\right)x^{A(z)/z}, \quad 0\leq x,z \leq 1$$ 
+>
+> where $A'(z)$ denotes the derivate of function $A(z)$ at point $z.$
+
+Since $A$ is a convex function defined on $[0, 1]$ and satisfies $-1 \leq A'(z) \leq 1$, by extension, we define $A'(1)$ as the supremum of $A'(z)$ over $(0, 1)$. By setting $x = 1$ in the previous result, we obtain the marginal distribution of $Z$:
+$$P(Z \leq z) = G_Z(z) = z + z(1 - z) \frac{A'(z)}{A(z)}, \quad 0 \leq z \leq 1.$$
+
+This result was demonstrated by Deheuvels (1991) [deheuvels1991limiting](@cite) in the case where $A$ admits a second derivative.
+
+
+## Simulation of Bivariate Extreme Value Distributions
```

Comment:
The two algorithms are not correctly referenced: I did not understand where they are coming from ? Could you cite the papers correcponding to them ? 

Moreover, it is unclear which one is actually implemented in the package : could you be a bit more precise in the wording here ?